templates/download/amd-xilinx/tab-1.html

```diff
@@ -1,151 +1,167 @@
-<div tabindex="0" role="tabpanel" id="kira-kv260-vision-ai-tab" aria-labelledby="kira-kv260-vision-ai">
-  <div class="row u-vertically-center u-no-padding">
-    <div class="col-6">
-      <h2>Download Ubuntu Desktop</h2>
-      <h3 class="p-heading--4">Ubuntu Desktop 22.04 LTS</h3>
-      <p class="u-sv1">The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
-      <p>
-        <a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x07-20230302-63.img.xz">Download 64-bit</a>
-      </p>
-      <p>Works on:</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">KR260 Robotics Starter Kit</li>
-        <li class="p-list__item is-ticked">KV260 Vision AI Starter Kit</li>
-      </ul>
+<div tabindex="0" role="tabpanel" id="kira-kv260-vision-tab" aria-labelledby="kira-kv260-vision" aria-hidden="true">
+  <div class="row p-strip is-shallow u-no-padding--top">
+    <div class="col-3 col-medium-2">
+      <h2 class="u-hide--small">Kria SOMs <br class="u-hide--medium" />(KR260/<br class="u-hide--medium" />KV260)</h2>
     </div>
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/08fdc6b3-xilinx-board.png",
+    <div class="col-6 amd-xilinx-container u-align--center col-medium-4">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/62c9deff-AMD%20KR260%20KV260.png",
         alt="AMD-Xilinx Kria KR260 circuit board",
-        width="312",
+        width="544",
         height="250",
         hi_def=True,
         loading="lazy"
         ) | safe
       }}
     </div>
   </div>
-  <div class="u-fixed-width">
-    <h4>First time installing Ubuntu on AMD-Xilinx?</h4>
-    <p>Please see the <a href="https://www.xilinx.com/KR260-Start">Kria KR260 Getting Started Guide for Ubuntu 22.04</a> for more information.</p>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
+    <div class="col-3 col-medium-2">
+      <h3>Ubuntu Desktop 22.04 LTS</h3>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p>Works on:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">KR260 Robotics Starter Kit</li>
+        <li class="p-list__item is-ticked">KV260 Vision AI Starter Kit</li>
+        <li class="p-list__item is-ticked">Please ensure that your Kria platform has the latest boot FW. FW updates and instructions are available on the <a href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/1641152513/Kria+K26+SOM">AMD-Xilinx K26 Wiki</a>.</li>
```

Comment:
Sorry, not sure why my comment didn't go through on this one, but this list item should be separated from the rest 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/65df686b-4e45-47eb-825e-c323579c495f)